### DISCUSSION TITLE
[@types/picomatch] Missing picomatch/posix types

### DISCUSSION THREAD

**Discussion by saperio** [DefinitelyTyped/DefinitelyTyped#70283] [Issues with a @types package]
Picomatch [exposes](https://github.com/micromatch/picomatch#picomatch-1) picomatch/posix with same types as picomatch, but no types for this import.

[a comment by an automated account is not shown]

**Comment by p-kuen** [comment]
The relevant file could be found here:
https://github.com/micromatch/picomatch/blob/master/posix.js

A PR for this would be welcome!
